recipe_scrapers/goodhousekeeping.py

```diff
@@ -0,0 +1,93 @@
+# mypy: allow-untyped-defs
+
+import re
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class GoodHousekeeping(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "goodhousekeeping.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".ingredients-body h3",
+            ".ingredient-lists li",
+        )
+
+    def instructions(self):
+        directions = self.soup.find(
+            "ul",
+            class_="directions",
+        )
+
+        methods = [
+            directions.find_all("p"),
+            directions.find("li", recursive=False).find_all("li"),
+        ]
+
+        for m in methods:
+            if len(m) != 0:
+                instructions = m
+                break
+
+        return "\n".join(
+            [
+                re.sub(r"Step \d", "", normalize_string(instruction.get_text())).strip()
+                for instruction in instructions
+            ]
+        )
+
+    def ratings(self):
+        return ""
```

Comment:
Hi @jpts - thank you for contributing!  I'm starting some code review now and will leave a few comments.

For the `ratings` field here: generally the house style is to return `None` (an empty / absence-of-a value) when we can't find matching content on the page.  In Python, that's distinct from `""` (the empty string).  However in this case, since we never return `ratings` at all, the easiest fixup is likely to remove the `ratings` method.